packages/aws-fargate/images/test-images/build-and-push-all.sh

```diff
@@ -1,39 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I rarely used this anymore, I think it is better to delete it than to keep maintaining it (or letting it become unmaintained).